src/components/common/Header.js

```diff
@@ -119,7 +119,7 @@ const Header = () => {
 
   const handleClick = (event, value) => {
     if (event) {
-      if (value === COMPOSITION) {
+      if (value.toLowerCase() === COMPOSITION) {
```

Comment:
please refer to the constant.js file where a constant for the composition has been already defined.

Reply:
Modifications have been made to use constants from constant.js file